### ISSUE TITLE
Use arguments in GraphQL

### ISSUE THREAD

**Issue by sanderpotjer** [page-8/craft-manytomany#41]
### Describe the bug

I can't seem to use arguments for the many-to-many field via GraphQL? The part `(limit: 3)` of the example query results in an error. Without 
```
query example {
  entry: artistsEntries(slug: "jackson-michael") {
    ... on artists_artist_Entry {
      title
      summary
      tracks (limit: 3) {
        title
      }
    }
  }
}
```
Unknown argument \"limit\" on field \"tracks\" of type \"artists_artist_Entry\". 

### Steps to reproduce

1. Setup Many-to-Many field (in this example `tracks`)
2. Retrieve data via GraphQL, e.g.
```
query example {
  entry: artistsEntries(slug: "jackson-michael") {
    ... on artists_artist_Entry {
      title
      summary
      tracks {
        title
      }
    }
  }
}
```
3. Confirm it works
4. Add limit for many to many field:
```
query example {
  entry: artistsEntries(slug: "jackson-michael") {
    ... on artists_artist_Entry {
      title
      summary
      tracks {
        title
      }
    }
  }
}
```
5. Error is thrown

### Craft CMS version

4.5.6.1

### Plugin version

dev-craft-4

### Multi-site?

Yes

**Comment by engram-design** [comment]
Should be fixed in [3.0.4](https://github.com/verbb/many-to-many/releases/tag/3.0.4)

**Comment by sanderpotjer** [comment]
@engram-design great, thanks! Confirm it's fixed.
